benchmarks/aspell_aspell_fuzzer/build.sh

```diff
@@ -15,12 +15,7 @@
 #
 ################################################################################
 
-# Suppress an immediate UBSan violation that prevents fuzzing
```

Comment:
Why is this not necessary anymore?

Reply:
I changed its build file to be [the same as in OSS-Fuzz](https://github.com/google/oss-fuzz/blob/master/projects/aspell/build.sh).

Reply:
ACK